packages/extension/app/v2/background.js

```diff
@@ -107,6 +107,11 @@ const connect = function (host, path, extraOpts) {
     return Promise.try(() => {
       return automation[method].apply(automation, args.concat(respond))
     }).catch((err) => {
+      ws.emit('automation:push:request', 'debug', {
```

Comment:
Is this temporary?

Reply:
Yep, missed removing it with the previous commit. Thanks for the catch!